src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -47,8 +47,9 @@ public abstract class ClientApplicationBase
 
         static ClientApplicationBase()
         {
-            PlatformPlugin.Logger.Information(null,
-                string.Format(CultureInfo.InvariantCulture,
+            RequestContext requestContext = new RequestContext(Guid.Empty);
```

Comment:
We can remove this static block and move this log statement to constructor ClientApplicationBase. this will make the log appear each time app object is created and thats a good thing.